docs/docs/order/company.md

```diff
@@ -43,6 +43,44 @@ The list of contacts associated with a particular company is available in the <s
 
 A *contact* can be assigned to orders, (such as [purchase orders](./purchase_order.md) or [sales orders](./sales_order.md)).
 
+### Addresses
+
+A company can have multiple registered addresses for use with all types of orders.
+An address is broken down to internationally recognised elements that are designed to allow for formatting an address according to user needs.
+Addresses are composed differently across the world, and Inventree reflects this by splitting addresses into components:
+- Line 1: Main street address
+- Line 2: Extra street address line
+- Postal Code: Also known as ZIP code, this is normally a number 3-5 digits in length
+- City: The city tied to the postal code
```

Comment:
Suggestion: In parts of Europe this field is sometimes referred to as Region

Reply:
Referring to City or Province?

Reply:
City